templates/client-auth-n.cfg.xml.erb

```diff
@@ -14,13 +14,21 @@
   <%- end -%>
 <%- end -%>
 <%- if @ignore_tenant_roles -%>
-    <ignore-tenant-roles>
+        <ignore-tenant-roles>
```

Comment:
@shintasmith  said "i changed existing indentation of existing elements, cuz the resulting xml was not indented properly" which makes me happy